src/main/resources/representative-details/translation.json

```diff
@@ -1,60 +1,62 @@
-          "organisationNameLabel": "Enw’r sefydliad neu’r busnes",
-          "contactNumberLabel": "Rhif cyswllt",
-          "emailAddressLabel": "Cyfeiriad e-bost"
-        }
+  "translations": {
+    "en": {
+      "continue": "Continue",
+      "serviceName": "Appeal to the First-tier Tribunal",
+      "title": "Representative's Details",
+      "line1": "Please enter details of the representative as stated on your tribunal form.",
+      "line2": "We will only use these details to contact this representative about this case.",
+      "fullNameLabel": "Full name",
+      "organisationNameLabel": "Organisation or business name",
+      "contactNumberLabel": "Contact number",
+      "emailAddressLabel": "Email address"
     },
-    "errors":{
-        "en": {
-            "representativeFullName": {
-                "required": "Please enter full name"
-            },
-            "representativeOrganisationName": {
-                "required": "Please enter organisation or business name"
-            },
-            "representativeContactNumber": {
-              "required": "Please enter a valid contact number",
-              "invalid": "Please enter a valid contact number"
-            },
-            "representativeEmailAddress": {
-              "required": "Please enter a valid email address",
-              "invalid": "Every email address consists of 3 elements: local-part, @ symbol (pronounced as “at”), and a domain name"
-            }
-        },
-        "cy": {
-          "representativeFullName": {
-            "required": "Nodwch enw llawn"
-          },
-          "representativeOrganisationName": {
-            "required": "Nodwch enw’r sefydliad neu fusnes"
-          },
-          "representativeContactNumber": {
-            "required": "Nodwch rif cyswllt dilys",
-            "invalid": "Nodwch rif cyswllt dilys"
-          },
-          "representativeEmailAddress": {
-            "required": "Nodwch gyfeiriad e-bost dilys",
-            "invalid": "Mae pob cyfeiriad e-bost yn cynnwys tair elfen: rhan lleol, symbol @ (a ynganir fel “at”), ac enw parth"
-          }
-        }
+    "cy": {
+      "continue": "Parhau",
+      "serviceName": "Apelio i’r Tribiwnlys Haen Gyntaf",
+      "title": "Manylion y cynrychiolydd",
+      "line1": "Rhowch fanylion y cynrychiolydd fel y’u nodir ar eich ffurflen tribiwnlys.",
+      "line2": "Byddwn ond yn defnyddio’r manylion hyn i gysylltu â’r cynrychiolydd ynghylch yr achos hwn.",
+      "fullNameLabel": "Enw llawn",
+      "organisationNameLabel": "Enw’r sefydliad neu’r busnes",
+      "contactNumberLabel": "Rhif cyswllt",
+      "emailAddressLabel": "Cyfeiriad e-bost"
     }
+  },
+  "errors": {
+    "en": {
+      "representativeFullName": {
+        "required": "Please enter full name",
+        "invalid": "Please enter a valid name"
+      },
+      "representativeOrganisationName": {
+        "required": "Please enter organisation or business name"
```

Comment:
Why there is no invalid message for organisation name? As the value will be entered by the users, we might need a validation check as like full name (if not similar)

Reply:
We assumed this was because an organisation name could include things like lowercase and numbers or maybe even special characters. The regex could end up being too broad. See: https://companieshouse.blog.gov.uk/2019/02/14/symbols-and-characters-in-a-company-name/

Reply:
If there are no validation in ExUI, we can proceed  (better to have similar approach as like ExUI)

Reply:
There is no validation in ExUI on the org, so we can proceed without validation - we are currently consulting the PO on the representative's **name** however, as there is no validation for this on Submit case or on ExUI

Reply:
Got confirmation from PO that there's no validation required for the representative's name too